react/Newsletter.tsx

```diff
@@ -109,14 +110,14 @@ function Newsletter(props: PropsWithChildren<Props>) {
       phone,
     }
 
-    const pixelEvent = customEventId
+    const pixelEvent: PixelData = customEventId
       ? {
           id: customEventId,
-          event: 'vtex:newsletterSubscription',
+          event: 'newsletterSubscription',
```

Comment:
@victorhmp This event type is not yet implemented, that`s why the PixelData type is acusing an error on this const.

I`ve already sent this PR to fix that.
https://github.com/vtex-apps/pixel-app-template/pull/15

Reply:
@Mogueno if you send us the link to the workspace you used to test and validate the feature, we can access it. If it's a b2b store, it's desirable that you provide us credentials or help us create our user.

Reply:
Also, you should rebase your code